modules/apps/jaggery/emm/src/emm/modules/policy.js

```diff
@@ -31,6 +31,63 @@ policyModule = function () {
     var publicMethods = {};
     var privateMethods = {};
 
+    privateMethods.getAllPoliciesOnError = function (responsePayload) {
```

Comment:
function name does not convey a proper meaning upon the actual usage of this function